packages/mvsp/mvsp-alpha.en.asciidoc

```diff
@@ -137,7 +140,9 @@ h| Description
   * Handling untrusted data. Example: Reusing data supplied by users within sensitive application contexts
 
 | 3.4 Time to fix vulnerabilities
-| Produce and deploy patches to address application vulnerabilities that materially impact security within 90 days of discovery
+| * Produce and deploy patches to address application vulnerabilities that materially impact security within 90 days of discovery
+  * For vulnerabilities with evidence of active exploitation, produce and deploy patches within 14 days of discovery
```

Comment:
In my view, although this control suggestion is logical, it introduces a sophisticated procedure for monitoring and pinpointing vulnerabilities designated as "actively exploited." This approach appears to deviate from the "minimality" principle of this guideline.

Reply:
I'm not sure that I agree fully here. Effectively we're already asking companies to maintain visibility into patches for dependencies (this is already a large ask). Adding a single reference for known exploited vulnerabilities doesn't seem like a significant burden.

As above, I don't think that we can ask companies in a minimum baseline to patch within 14 days. We can recommend that known exploited vulnerabilities are prioritised.